app/helpers/application_helper.rb

```diff
@@ -134,8 +134,24 @@ def yes_no_options
     [%w[Yes true], %w[No false]]
   end
 
+  def display_downtime_warning?
+    [Settings.downtime_warning_enabled?,
+     Date.current <= Settings.downtime_warning_date.to_date,
+     current_user,
+     on_home_page?].all?
+  end
+
   private
 
+  def on_home_page?
+    ['/',
+     '/super_admins',
+     '/case_workers',
+     '/case_workers/admin',
+     '/case_workers/claims',
+     '/external_users'].include?(request.path)
```

Comment:
This array could be moved into a frozen constant. Also, Rubocop would normally suggest changing this to

```suggestion
    %w[/
     /super_admins
     /case_workers
     /case_workers/admin
     /case_workers/claims
     /external_users].include?(request.path)
```

(See [Style/WordArray](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/WordArray))

I'm not sure why it is not flagging it here.